scripts/install.sh

```diff
@@ -1,12 +1,13 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Is there a reason for the switch to `bash` ?

Reply:
Or maybe @schnappfuesch can help 😊

Reply:
the `for-loop` is using an invalid bracket syntax (`((...))`) for `sh`. The `bash` is a `sh` extension that understand this format. The same is valid for square brackets in `bash` (`[[...]]`) and `sh` (`[...]`).

Reply:
A more portable solution would be
```suggestion
#!/usr/bin/env bash
```
which works on all systems (including the more obscure ones like NixOS, where `/bin/bash` does not exist)